wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WContainer.java

```diff
@@ -31,57 +26,37 @@ public void add(final WComponent component, final String tag) {
 		super.add(component, tag);
 	}
 
-	/**
-	 * {@inheritDoc}
-	 */
 	@Override
 	public WComponent getChildAt(final int index) {
 		return super.getChildAt(index);
 	}
 
-	/**
-	 * {@inheritDoc}
-	 */
 	@Override
 	public int getChildCount() {
 		return super.getChildCount();
 	}
 
 	@Override
-	/**
-	 * {@inheritDoc}
-	 */
 	public List<WComponent> getChildren() {
 		return super.getChildren();
 	}
 
-	/**
-	 * {@inheritDoc}
-	 */
 	@Override
 	public void remove(final WComponent child) {
 		super.remove(child);
 	}
 
-	/**
-	 * {@inheritDoc}
-	 */
 	@Override
 	public void removeAll() {
 		super.removeAll();
 	}
 
-	/**
-	 * {@inheritDoc}
-	 */
 	@Override
 	public int getIndexOfChild(final WComponent childComponent) {
 		return super.getIndexOfChild(childComponent);
 	}
 
 	/**
-	 * {@inheritDoc}
```

Comment:
Please retain this inheritDoc tag as the inheritance needs to be explicit due to the other comments in the block.